posthog/queries/trends/formula.py

```diff
@@ -72,7 +72,9 @@ def _run_formula_query(self, filter: Filter, team: Team):
             additional_values: Dict[str, Any] = {"label": self._label(filter, item)}
             if is_aggregate:
                 additional_values["data"] = []
-                additional_values["aggregated_value"] = item[1][0]
+                additional_values["aggregated_value"] = (
```

Comment:
Agree it's better not to explode. Though can we still capture this with Sentry with some context, so that we better know what might be causing this + how important it is to solve this?